site/getting-started.md

```diff
@@ -17,15 +17,39 @@ Before you start you will need:
 
 ## Add Contour to your cluster
 
-Run:
+### Option 1
+
+Deploy Contour using the rendered manifest:
 
 ```bash
 $ kubectl apply -f {{ site.url }}/quickstart/contour.yaml
 ```
 
+### Option 2
```

Comment:
Is the operator still marked experimental/not-GA? If so, we probably should call it out here, until we are ready to call the operator GA.